dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateConnectionTask.java

```diff
@@ -32,6 +32,7 @@ public static Session createConnection(final NewSessionModel newSessionModel, fi
                             new Credentials(newSessionModel.getAccessKey(), newSessionModel.getSecretKey()))
                     .withHttps(newSessionModel.isUseSSL()).withCertificateVerification(false)
                     .withProxy(newSessionModel.getProxyServer())
```

Comment:
Many many files are missing the copyright comment.  We need to create a task to add it to all files